plugins/prefect/access_key.go

```diff
@@ -0,0 +1,56 @@
+package prefect
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AccessKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.AccessKey,
+		DocsURL:       sdk.URL("https://docs.prefect.io/2.10.13/cloud/users/api-keys/"),
```

Comment:
Is there a specific reason we're referring to the docs for 2.10.13? Prefect seems to redirect to the latest version if version is left out of the URL:
```suggestion
		DocsURL:       sdk.URL("https://docs.prefect.io/cloud/users/api-keys/"),
```

Reply:
Prefect make a lot of changes and their documentation is not 100% trustworthy (although they are becoming better at it) so I wanted to make sure the plugin referenced the documentation version used. But I have applied your suggestion